xcell/tests/test_mapper_p18cmbk.py

```diff
@@ -30,9 +41,16 @@ def test_get_signal_map():
 
 
 def test_get_mask():
-    m = get_mapper()
+    c = get_config()
+    c['path_lite'] = './lite'
+    m = xc.mappers.MapperP18CMBK(c)
     d = m.get_mask()
     assert np.all(np.fabs(d-1) < 1E-5)
+    # Now read from lite path
+    m2 = xc.mappers.MapperP18CMBK(c)
+    d2 = m2.get_mask()
+    assert np.all(np.fabs(d-d2) < 1E-10)
+    os.system('rm -r ./lite')
```

Comment:
I find safer (and more compatible with different OS): `shutil.rmtree(./lite)`

Reply:
Sounds good. Done!